src/Microsoft.ML.Transforms/NAIndicatorTransform.cs

```diff
@@ -368,8 +368,8 @@ private void FillValues(int srcLength, ref VBuffer<DvBool> dst, List<int> indice
 
                 int ii = 0;
                 // Assigns values correctly depending on the sense.
-                DvBool hit = sense ? DvBool.True : DvBool.False;
-                DvBool miss = sense ? DvBool.False : DvBool.True;
+                bool hit = sense;
```

Comment:
>bool hit = sense; [](start = 16, length = 17)

is it necessary? can you just use sense instead of hit? (I'm totally fine with miss)